models/src_amazon_ads.yml

```diff
@@ -230,7 +230,7 @@ sources:
           - name: campaign_id
             description: "{{ doc('campaign_id') }}"
           - name: asin
-            description: The ASIN associated with the ad.
+            description: The ASIN associated with the ad. 
```

Comment:
Can you add more clarity to what ASIN is?

Reply:
done.